catalog/spatial/registry/pom.xml

```diff
@@ -157,6 +157,6 @@
         <module>registry-federation-admin-impl</module>
         <module>registry-app</module>
         <module>registry-admin-modules</module>
-
+        <module>registry-report-viewer</module>
```

Comment:
@vina will be putting her stuff in the same bundle (I think) so it will have the viewer.